recipes/sides/freaky-freekeh.md

```diff
@@ -0,0 +1,33 @@
+# Freaky Freekeh
+
+![Difficulty Badge](https://img.shields.io/badge/Difficulty-20%25-green.svg) 
+![Speed Badge](https://img.shields.io/badge/Speed-0.3hrs-brightgreen.svg)
+![Vegetarian Friendly Badge](https://img.shields.io/badge/Vegetarian-True-brightgreen.svg)
+![Vegan Friendly Badge](https://img.shields.io/badge/Vegan-True-brightgreen.svg)
+![Cost Badge](https://img.shields.io/badge/Cost-Cheap-green.svg)
+![Serves Badge](https://img.shields.io/badge/Serves-8-green.svg)
+![Reception Badge](https://img.shields.io/badge/Reception-Positive-green.svg)
+![Green Badge Count Badge](https://img.shields.io/badge/Green%20badge%20count-100%25-green.svg)
```

Comment:
this is not badge that you should include by the (CONTRIBUTING.md)[https://github.com/foundersandcoders/nazareth-recipes/blob/master/CONTRIBUTING.md] guide

Reply:
it's not required but the guide says "Feel free to add other any badges you deem appropriate!"